packages/legend-query-builder/style/_query-builder-projection.scss

```diff
@@ -65,6 +65,7 @@
     width: 1.6rem;
     min-width: 1.6rem;
     margin-left: 0.5rem;
+    cursor: move;
```

Comment:
should be `grab/grabbing` instead of `move` - https://developer.mozilla.org/en-US/docs/Web/CSS/cursor ?